packages/modules/src/BatchedSessionRouterModule.ts

```diff
@@ -150,21 +154,21 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
       const proof = this.sessionKeyManagerModule.merkleTree.getHexProof(
         ethers.utils.keccak256(leafDataHex) as unknown as Buffer
       )
-      proofs.push(proof)
+
+      sessionDataTuple.push(proof)
+      sessionDataTuple.push('0x')
+
+      sessionDataTupleArray.push(sessionDataTuple)
     }
 
     // Generate the padded signature with (validUntil,validAfter,sessionVerificationModuleAddress,validationData,merkleProof,signature)
+
+    Logger.log('whole tuple ', sessionDataTupleArray)
```

Comment:
Should we change it to Logger.debug? Don't want to print any logs in normal mode.

Reply:
ok will change it.

Reply:
done